setup.py

```diff
@@ -156,8 +156,6 @@ def write_version(filename=os.path.join(*['airflow',
     'hmsclient>=0.1.0',
     'pyhive>=0.1.3',
     'impyla>=0.13.3',
-    'thrift_sasl==0.2.1',
```

Comment:
are we going to remove this dependency?

Reply:
@Fokko Are the dependency removals intentional?